src/search/SearchSuggestionItem.js

```diff
@@ -28,6 +29,13 @@ export const styles = theme => ({
       },
     },
   },
+  text: {
+    '& .highlight': {
```

Comment:
I think it would be more straightforward if highlight was a jss class rather than a css class.
```suggestion
    text: { },
    highlight: {
```